HelpAssistHer/help-assist-her
# Add community health center and pregnancy center icons

## Description
This PR adds the community health center and pregnancy center icons. It also adds the functionality to click a button and change the color.

## Test Plan
1. Go to http://localhost:4000/mini-app
1. See this:
<img width="556" alt="screen shot 2018-10-29 at 8 10 53 pm" src="https://user-images.githubusercontent.com/6378435/47693426-e4906800-dbb6-11e8-9831-f7e19a3b5e15.png">
3. Click one of the buttons and see that it changes color (even if you move the mouse away):
<img width="622" alt="screen shot 2018-10-29 at 9 12 15 pm" src="https://user-images.githubusercontent.com/6378435/47695484-7c924f80-dbbf-11e8-9b0c-e44d43322e1b.png">

Note: this is not exactly what the designs show, but to keep things simple for now, let's just change the background color instead of the icon. We can go back and make that better after the beta version is released.
